calico-cloud/visibility/elastic/flow/aggregation.mdx

```diff
@@ -10,96 +10,63 @@ Configure flow log aggregation level to reduce log volume and costs.
+depends on your deployment; we recommend reviewing aggregation levels to understand what information gets grouped (and thus suppressed from view).
 
 ### Aggregation types and levels
 
-The following table summarizes the aggregation levels by flow log traffic.
-
-| Flow log aggregation by... | Available for...                                | Aggregates all flows that share...                                                                                                                                                                                                                                                                     |
-| -------------------------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| Pod prefix                 | Allowed and denied traffic (default is allowed) | `FlowLogsFileAggregationKindForAllowed`<br />**0**, No aggregation <br />**1**, A source port on each node.<br />**2, (default)** Source ports, or are from the same ReplicaSet. <br />**3**, Destination and source ports, and are from the same ReplicateSet.                                        |
-| Source port                | Allowed and denied traffic                      | `FlowLogsFileAggregationKindForAllowed`<br/>`FlowLogsFileAggregationKindForDenied`<br />**0**, No aggregation<br />**1, (default)** Source port on each node.<br />**2**, Source ports, or are from the same ReplicaSet.<br />**3**, Destination and source ports, and are from the same ReplicateSet. |
+For allowed flows, the default aggregation level is 2, `AnyProcessInSamePodPrefix` and for denied flows the default aggregation level is 1,
+`AnyProcessInSameSourcePod`.
+
+The following table summarizes the aggregation levels by flow log traffic:
+
+| **Level** | **Name**                            | **Description**                                                   |
+|-----------|-------------------------------------|-------------------------------------------------------------------|
+| 0         |                                     | No aggregation                                                    |
+| 1         | AnyProcessInSameSourcePod           | Identity fields below source pod level are masked out. It means that if multiple processes or containers, within the same source pod, perform the same operation, the events are aggregated.       |
+| 2         | AnyProcessInSameSourcePodPrefix     | Identity fields below source pod-prefix level are masked out. It means that if multiple processes or containers, within pods with the same prefix, perform the same operation, the events are aggregated. |
```

Comment:
```suggestion
| 2         | AnyProcessInSameSourcePodPrefix     | In addition to the above, source pod names are aggregated based on their shared prefixes.  This means that flows, to the same destination, from pods within the same Deployment/ReplicaSet are aggregated together. |
```
"pod-prefix" is an implementation detail; I think it;d help to link back to user concepts such as ReplicaSets/Deployments

Reply:
Thanks for the suggestion, I've added it and adapted the level 3 description accordingly.